criu/ubi8/Dockerfile

```diff
@@ -92,19 +88,10 @@ RUN cd /tmp; \
     git clone -b ${CRIU_BRANCH} ${CRIU_REPO}; \
     mv criu criu-build; \
     cd criu-build; \
-    git config --global user.email "build@example.com"; \
-    git config --global user.name "build"; \
-    if [ "x${CRIU_SHA}" == x ] ; then \
-        CRIU_SHA=$(git rev-parse HEAD); \
-    fi; \
-    git reset --hard ${CRIU_SHA}; \
-    if [ "x${CRIU_FIX_SHA}" != x ] ; then \
-        git remote add dev ${CRIU_FIX_REPO}; \
-        git fetch dev; \
-        git cherry-pick ${CRIU_FIX_SHA}; \
+    if [ "x${CRIU_SHA}" != x ] ; then \
```

Comment:
Is this variable CRIU_SHA expected to be passed as an argument while building the docker image in any scenario ?

Reply:
No, is not. 
It is not mandatory. It is useful for automated builds. A Jenkins pipeline will pass it to all downstream jobs to ensure they all build same level across all platforms.

Reply:
Thanks for clarifying it.